static/pride.css

```
@@ -66,9 +66,18 @@
     border-left: 0;
 }
 
-/*#viewDropdownPlaceholder > div > ul > li:nth-child(10) {*/
-/*    display: none !important;*/
-/*}*/
+/* Hide 3D (NGL), Alignment, Legend & Colours, Spectrum views in PRIDE deployment */
+#viewDropdownPlaceholder > div > ul > li:nth-child(1),
+#viewDropdownPlaceholder > div > ul > li:nth-child(3),
+#viewDropdownPlaceholder > div > ul > li:nth-child(6),
+#viewDropdownPlaceholder > div > ul > li:nth-child(9) {
+    /*display: none !important;*/
```

Stylelint (17.4.0)

[error] Expected whitespace after "/*" (comment-whitespace-inside)

(comment-whitespace-inside)